src/pages/academy/_recent-featured-posts.js

```diff
@@ -81,8 +81,9 @@ const RecentFeaturedPosts = ({ recent_data, featured_data }) => {
                                         {headline_recent?.published_date &&
                                             convertDate(headline_recent?.published_date)}
                                         <ClockIcon src={Clock} />
-                                        {headline_recent.read_time_in_minutes}{' '}
-                                        <Localize translate_text="min read" />
+                                        {calculateReadTime(stripHTML(headline_recent?.blog_post)) +
+                                            ' ' +
+                                            localize('min read')}
```

Comment:
Can we directly return this string from `calculateReadTime` function?

Reply:
Yep, that'll be better. I will shift `stripHTML` in there as well since they are all using it.